seqr/views/utils/file_utils.py

```diff
@@ -37,7 +42,7 @@ def save_temp_file(request):
 
 def parse_file(filename, stream):
     if filename.endswith('.tsv') or filename.endswith('.fam') or filename.endswith('.ped'):
-        return [map(lambda s: s.strip().strip('"'), line.rstrip('\n').split('\t')) for line in stream]
+        return [[s.strip().strip('"') for s in line.rstrip('\n').split('\t')] for line in stream]
 
     elif filename.endswith('.csv'):
         return [row for row in csv.reader(stream)]
```

Comment:
this is where you would do the `TextIOWrapper ` conversion

Reply:
I'm not sure it can be done here or not because TextIOWrapper wraps BytesIO and it seems difficult to know if the input stream is a BytesIO or not. But, I will have a try to move the TextIOWrapper.

Reply:
if it sis a stream and you wrap it in `TextIOWrapper` that should be fin. And if it is a string and you wrap it in `TextIOWrapper` that should also be fine. So i'm not sure what the issue is